.github/config.yml

```diff
@@ -0,0 +1,27 @@
+# https://github.com/behaviorbot/welcome
+# combines new-issue-welcome, new-pr-welcome, and first-pr-merge
+
+# Comment on first-time issues
+# https://github.com/behaviorbot/new-issue-welcome
+newIssueWelcomeComment: >
+  👋 Thanks for opening your first issue here! If you're reporting a bug, please make sure you include steps to reproduce it. We get a lot of issues on this repo, so please be patient and we will get back to you as soon as we can.
+
+  To make it easier for us to investigate your issue, please follow the [contributing guidelines](https://github.com/nodejs/node/blob/master/doc/guides/contributing/issues.md).
```

Comment:
It would be helpful to list common `wrong repo` cases here (e.g. `npm`, `v8`, `node-gyp`, .etc) but we can do that in a future PR.